modules/library/render/src/main/java/org/geotools/renderer/lite/RenderingTransformationHelper.java

```diff
@@ -110,7 +110,9 @@ public Object applyRenderingTransformation(Expression transformation,
                     final GridCoverage2DReader reader = (GridCoverage2DReader) GRID_PROPERTY_NAME.evaluate(gridWrapper);
                     // don't read more than the native resolution (in case we are oversampling)
                     if(CRS.equalsIgnoreMetadata(reader.getCoordinateReferenceSystem(), gridGeometry.getCoordinateReferenceSystem())) {
-                         MathTransform g2w = reader.getOriginalGridToWorld(PixelInCell.CELL_CENTER);
+                        //GEOS-8070, changed the pixel anchor to corner. CENTER can cause issues with
+                        //the BBOX calculation and cause it to be incorrect
+                         MathTransform g2w = reader.getOriginalGridToWorld(PixelInCell.CELL_CORNER);
```

Comment:
It is consistent with other places where we do coverage rendering (they are using CELL_CORNER, not CELL_CENTER), I'm good with it.